tsconfig.json

```diff
@@ -26,5 +27,5 @@
 
     "forceConsistentCasingInFileNames": true /* Disallow inconsistently-cased references to the same file. */
   },
-  "exclude": ["node_modules", "webpack.config.js"]
+  "exclude": ["node_modules", "webpack.config.js", "**/*.js"]
```

Comment:
This seems to be the thing that makes the compile succeed. Keep this and turn Quill back on everywhere, and it should work.